packages/widget-message-meet/src/demo/demo.js

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable react/no-set-state, global-require */
```

Comment:
Oh, I was using your code for the background image at one point, but since I'm not now, I'll remove the rule disabling.